server/gitlab/api.go

```diff
@@ -14,9 +14,13 @@ import (
 const (
 	stateOpened      = "opened"
 	scopeAll         = "all"
-	withLabelDetails = true
 )
 
+type GitlabMergeRequest struct {
```

Comment:
We already have a struct defined with the name `MergeRequest` in the `link_tooltip` branch. Make sure we use them before the final merge in matter most.